### DISCUSSION TITLE
How to disable 'mqtt-subscription-" queue created automatically?

### DISCUSSION THREAD

**Discussion by chienhust** [rabbitmq/rabbitmq-server#7762] [Questions] [answered]
When i create a connection between MQTTNet and RabbitMQ Server, i see that queue named "mqtt-subscription-57e95593-3e68-4dbe-a7d7-fecf1fece4c2qos0" that is created automatically on RabbitMQ, so How to disable 'mqtt-subscription-" queue created automatically? Thank you very much.

**Comment by michaelklishin** [comment, marked as answer]
Those queues are created by the MQTT plugin when an MQTT client subscribes to a topic. You can disable the plugin but then you won't be able to use MQTT with this RabbitMQ installation.

**Reply by chienhust** [reply]
Thank you for your reply, I have to use MQTT plugin for MQTT connection, is there other way to remove/hide "mqtt-subscription-" queues created automatically? because I don't need to work with these queues.

**Reply by mkuratczyk** [reply]
The queues are created because your client applications subscribe to MQTT topics and RabbitMQ MQTT plugin implementats subscription as queues: https://www.rabbitmq.com/mqtt.html#implementation. The best way to not have these queues is to not subscribe to topics if you don't need to be subscribed.

**Reply by chienhust** [reply]
@mkuratczyk Thank you very much.

**Comment by m-arre** [comment]
If you're using the management UI you could hide them by setting this regex in the search box `^((?!mqtt-subscription-*).)*$`

**Reply by chienhust** [reply]
@m-arre  Thank you very much.
